scripts/validate_rst_title_capitalization.py

```diff
@@ -140,14 +139,18 @@
     "Panel",
 }
 
-CAP_EXCEPTIONS_DICT = {word.lower(): word for word in CAPITALIZATION_EXCEPTIONS}
+CAP_EXCEPTIONS_DICT = {
+    exception.lower() if type(exception) is str else exception[0].lower(): exception
```

Comment:
```suggestion
    exception.lower() if isinstance(exception, str) else exception[0].lower(): exception
```

But probably better:
```suggestion
    exception[0].lower() if isinstance(exception, collections.abc.Sequence) else exception.lower(): exception
```

And better keep word, or something similar. even if it's not as accurate, it's better than exception (which in Python is `ValueError`...)